ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -304,7 +322,7 @@ open class SearchBar: UIView {
         searchTextFieldBackgroundViewTrailingConstraint = searchTextFieldBackgroundViewTrailing
         constraints.append(searchTextFieldBackgroundViewTrailing)
 
-        //search icon container
+        /// search icon container
```

Comment:
>/// [](start = 8, length = 3)

revert

Reply:
done

---
In reply to: [467852130](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r467852130) [](ancestors = 467852130)